# sets maximal velocity and acceleration values along the trajectory
dynamic_params: {
  max_v: 5,
  max_a: 2.3,
  max_v_rot: 2,
  max_a_rot: 2,
  min_length: 5,
  max_groups: 20,
  tol_dis: 2,
  motion: 1,
}

waypoints: {

  directionGoals_x: [50, 100, 150,  150,  60,  0,    0,    60,    120,  150,],
  directionGoals_y: [-5,   0,   0,  -50, -50, -50,  -120, -120,  -120, -150,],
  const_height: 10,
  DilationRadius: 1,

  pos_waypt_x: [1,4,7], 
  pos_waypt_y: [0,0,0],
  pos_waypt_z: [7,7,7],  

  vel_waypt_x: [0,0,0],
  vel_waypt_y: [0,0,0],
  vel_waypt_z: [0,0,0,],

  acc_waypt_x: [0,0,0],
  acc_waypt_y: [0,0,0],
  acc_waypt_z: [0,0,0],

}